### PR TITLE
Fix pre-commit hook config

### LINKED ISSUE #163: BUG: if used with pre-commit and additional args -> Invalid value for "--msg-filename"
When I use gitlint with pre-commit and want to add `--extra-path` for custom rules like so:

```yaml
repos:
  - repo: https://github.com/jorisroovers/gitlint
    rev: c359748723a52f6c0c5f87c3b71749f75179ca74  # frozen: v0.14.0
    hooks:
      - id: gitlint
        args: [--extra-path pre-commit-helper.py]
```

I get the error:

```console
gitlint..................................................................Failed
- hook id: gitlint
- exit code: 253

Usage: gitlint [OPTIONS] COMMAND [ARGS]...
Try "gitlint --help" for help.

Error: Invalid value for "--msg-filename": Could not open file: --extra-path pre-commit-helper.py: No such file or directory

*** Exit Code: 1 ***
```

Reason for this is that my added args `[--extra-path pre-commit-helper.py]` are appended to the end after the entry-point. In gitlint's `.pre-commit-hooks.yaml` the entry contains args and `--msg-filename` as one expects a value to follow.

As a workaround you have to overwrite the entry in `.pre-commit-config.yaml` like so:

```yaml
  - repo: https://github.com/jorisroovers/gitlint
    rev: c359748723a52f6c0c5f87c3b71749f75179ca74  # frozen: v0.14.0
    hooks:
      - id: gitlint
        entry: gitlint
        args: [--extra-path, pre-commit-helper.py, --staged, --msg-filename]
```

As a fix you have to move the args from `entry` to `args`. I will submit a PR.
Sure you then have to specify them again, if you overwrite them, but that gives you more granular control as a user and its cleaner IMHO.

### PR DESCRIPTION
**Pull request by Cielquan**
fixes #163

### CODE REVIEW

**Comment by jorisroovers** [comment]
Tests fail on gitlint checks
```sh
3: B5 Body message is too short (10<20): "fixes #163"
```

**Comment by Cielquan** [comment]
> Tests fail on gitlint checks
> 
> ```shell
> 3: B5 Body message is too short (10<20): "fixes #163"
> ```

Should be fixed now.

File: docs/index.md [hunk 1]
```diff
@@ -219,11 +219,14 @@ your `.pre-commit-config.yaml` file like so:
     rev:  # Fill in a tag / sha here
     hooks:
     -   id: gitlint
-        stages: [commit-msg]
```
Comment (Cielquan) [reply docs/index.md:222]: Yes if the upstream repo defines the `stages` then the user does not have to, if he/she does not wish to change them.

**Comment by jorisroovers** [comment]
I did't try this out myself, maybe later - I'll take your word for it :-)

Thanks again for the PR!

**Comment by Cielquan** [comment]
> I did't try this out myself, maybe later - I'll take your word for it :-)
> 
> Thanks again for the PR!

Thanks for merging :)

File: docs/index.md (hunk 1 above)
Comment (jorisroovers) [inline docs/index.md:222]: Is `stages` not required? Does it get taken automatically from the upstream repo's `.pre-commit-hooks.yaml`?

**Pull request merged**